### PR TITLE
SSL update for legacy hosts.

### PR DESCRIPTION
**Pull request by kennethgillen**
A playbook to deploy SSL certs to legacy hosts.

Supporting MT configuration, including an inventory update at: https://github.com/openmicroscopy/management_tools/pull/864

### CODE REVIEW

File: legacy/ssl-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+# Playbook which deploys SSL certificates to legacy
+# (not entirely Ansible managed) hosts
+
+# openmicroscopy.org wildcard
+- hosts: legacy-ssl-org-wildcard
+  roles:
+  - role: openmicroscopy.ssl-certificate
+    # Vars here since vars will change for each certificate we're deploying.
+    ssl_certificate_public_content: "{{ ssl_org_wildcard_certificate_public_content }}"
```
Comment (manics) [inline legacy/ssl-deploy.yml]: Optional: You could avoid having multiple variables by omitting this variable from the playbook and defining it in host/group vars only, e.g.:
- https://github.com/openmicroscopy/management_tools/blob/594a39d85dfd25869382a9da0b409a8c901a0adf/idr/inventory/group_vars/proxy-hosts.yml#L5

Comment (kennethgillen) [reply legacy/ssl-deploy.yml]: I envisage a second instance of this same role in this same playbook, for the `.org.uk` wildcard.

So in this case if `ssl_certificate_public_content` was defined in two hostgroups, and the same host was in both groups, I don't know if Ansible would cope with that. i.e. how would it know which to use for each invocation? Or is there obvious way to work around/apply that pattern that I've missed?

**Comment by kennethgillen** [comment]
Deployed to `ci-hosts`: 

```
PLAY RECAP **************************************************************************************************
cowfish.openmicroscopy.org : ok=10   changed=4    unreachable=0    failed=0
octopus.openmicroscopy.org : ok=10   changed=4    unreachable=0    failed=0
ray.openmicroscopy.org     : ok=10   changed=4    unreachable=0    failed=0
seabass.openmicroscopy.org : ok=10   changed=4    unreachable=0    failed=0
trout.openmicroscopy.org   : ok=0    changed=0    unreachable=1    failed=0
```

**Comment by kennethgillen** [comment]
This is ready to merge.

**Pull request merged**